interfaces/builtin/modem_manager.go

```diff
@@ -1249,7 +1249,7 @@ func (iface *modemManagerInterface) DBusPermanentSlot(spec *dbus.Specification,
 
 func (iface *modemManagerInterface) UDevPermanentSlot(spec *udev.Specification, slot *snap.SlotInfo) error {
 	spec.AddSnippet(modemManagerPermanentSlotUDev)
-	spec.TagDevice(`KERNEL=="tty[A-Z]*[0-9]*|cdc-wdm[0-9]*"`)
+	spec.TagDevice(`KERNEL=="tty[a-zA-Z].*|cdc-wdm[0-9]*"`)
```

Comment:
I'm sorry, I used a shorthand and didn't mean for it to be used literally. Please use:
```
spec.TagDevice(`KERNEL=="tty[a-zA-Z]*[0-9]*|cdc-wdm[0-9]*"`)
```

Reply:
It is important to use `KERNEL=="tty[a-zA-Z]*[0-9]*|cdc-wdm[0-9]*"` because this is not regex, but shell globbing rules (see https://www.freedesktop.org/software/systemd/man/udev.html). We don't want to match /dev/tty or /dev/ttyprintk, so you need to keep the `[0-9]` in there.